lib/model/CEventRateModel.cc

```diff
@@ -584,6 +581,11 @@ void CEventRateModel::fill(model_t::EFeature feature,
                            bool interim,
                            CProbabilityAndInfluenceCalculator::SParams& params) const {
     const TFeatureData* data{this->featureData(feature, pid, bucketTime)};
+    if (data == nullptr) {
+        LOG_ERROR(<< "Feature data unexpectedly null");
```

Comment:
The other logging for null feature data is at the `TRACE` level.  I think that's OK, on the assumption that the `featureData` method itself always logs at the `ERROR` level before returning `nullptr`.  So I think this should be `TRACE` too.  (In fact, the places in the code that already did null checks on the return value of `featureData` don't log at all - they just continue/return.  `TRACE` is equivalent for production code as we compile it out.)